util/docker/docker.go

```diff
@@ -23,7 +23,7 @@ var (
 	errRunFailed             = errors.NewKind("cannot run bblfshd container")
 	errConnectToDockerFailed = errors.NewKind("could not connect to docker")
 	errResourceStartFailed   = errors.NewKind("could not start resource")
-	errPortWaitTimeout       = errors.NewKind("could not wait until port %d is enabled")
+	errPortWaitTimeout       = errors.NewKind("could not wait until port %s is enabled")
```

Comment:
just in case I suggest to use `%v`, so you can pass string or number here and it will work.

Reply:
Done